docs/dev-env-setup.md

```diff
@@ -180,32 +182,31 @@ copy config/secrets.py.example config/secrets.py
 
 You can now edit `secrets.py` and
 
-1. You have the option of creating a "bot account" (a dummy account for the purpose of testing), or using your own account for local development. Either way, you'll need a personal access token to proceed &mdash; this is the oauth token we use to report issues on behalf of people who don't want to give GitHub OAuth access (or don't have GitHub accounts).
-
-  The [instructions for creating a personal-access token](http://help.github.com/articles/creating-an-access-token-for-command-line-use) are given on GitHub. Select public_repo to grant access to the public repositories through the personal access token.  Once you have created the token you can add it in the variable `OAUTH_TOKEN = ""`.
+1.  You have the option of creating a "bot account" (a dummy account for the purpose of testing), or using your own account for local development. Either way, you'll need a personal access token to proceed &mdash; this is the oauth token we use to report issues on behalf of people who don't want to give GitHub OAuth access (or don't have GitHub accounts).
 
-3. Add values for `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET`. If you're part of the [webcompat GitHub organization](https://github.com/webcompat), you can [get the client ID and client secret from GitHub](https://github.com/organizations/webcompat/settings/applications/). Otherwise, create your own local application ([instructions here](https://github.com/settings/applications/new)) &mdash; when prompted for an "Authorization callback URL", use `http://localhost:5000/callback`, and take note of the client ID and client secret GitHub give you. 
+The [instructions for creating a personal access token](http://help.github.com/articles/creating-an-access-token-for-command-line-use) are given on GitHub. Select public_repo to grant access to the public repositories through the personal access token. Once you have created the token you can add it in the variable `OAUTH_TOKEN = ""`.
 
-> Note: Cloud 9 users should use `http://yourapp.c9users.io:8000/callback`for the Authorization callback URL instead
+2.  Add values for `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET`. Create your own local application ([instructions here](https://github.com/settings/applications/new)) &mdash; when prompted for an "Authorization callback URL", use `http://localhost:5000/callback`, and take note of the client ID and client secret GitHub gives you.
 
-  When you have the client ID and client secret, put them in the corresponding lines in `secrets.py`:
+> **Note**: Cloud 9 users should use `http://yourapp.c9users.io:8000/callback` for the Authorization callback URL instead.
```

Comment:
Same thing here. 
```suggestion
    > **Note**: Cloud 9 users should use `http://yourapp.c9users.io:8000/callback` for the Authorization callback URL instead.
```

and so on.

Reply:
I think I got all of the indents under the numbered list. If I missed anything, LMK.